test/lib/eventFactory.test.js

```diff
@@ -775,7 +770,7 @@ describe('Event Factory', () => {
             }, {
                 id: 2,
                 pipelineId: 8765,
-                name: 'PR-1:test',
+                name: 'PR-1:publish',
```

Comment:
`PR-1:publish` is proper job name if trigger `startFrom` is `~pr`.
https://github.com/screwdriver-cd/models/blob/master/test/lib/eventFactory.test.js#L207